tox.ini

```diff
@@ -0,0 +1,13 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27
```

Comment:
This should probably be for python3.5 as home assistant uses python3+. Although since there are no tests I would guess that tox.ini might not be useful anyway.